### PR TITLE
Chan settings: renamed 'buffer' to 'buffer size'

### PR DESCRIPTION
**Pull request by ChasedSpade**
Changed the 'buffer' variable in SETCHAN and GETCHAN to match the 'buffercount' variable used in SET.

### CODE REVIEW

**Comment by jpnurmi** [comment]
In master, there's a separate user-level setting for channel and query buffer sizes (#967). They are called ```ChanBufferSize```and ```QueryBufferSize```, respectively. If the chan-level setting is renamed, a) it should follow the same naming convention, and b) backwards compatibility should be kept like #998 did.

File: modules/controlpanel.cpp [hunk 1]
```diff
@@ -705,12 +705,12 @@ class CAdminMod : public CModule {
 		for (CChan* pChan : vChans) {
 			if (sVar == "defmodes") {
 				PutModule(pChan->GetName() + ": DefModes = " + pChan->GetDefaultModes());
-			} else if (sVar == "buffer") {
+			} else if (sVar == "buffercount") {
```
Comment (jpnurmi) [inline modules/controlpanel.cpp]: ```} else if (sVar == "buffersize" || sVar == "buffer") {```
to keep backwards compatibility

File: modules/controlpanel.cpp [hunk 2]
```diff
@@ -763,11 +763,11 @@ class CAdminMod : public CModule {
 			if (sVar == "defmodes") {
 				pChan->SetDefaultModes(sValue);
 				PutModule(pChan->GetName() + ": DefModes = " + sValue);
-			} else if (sVar == "buffer") {
```
Comment (jpnurmi) [inline modules/controlpanel.cpp]: ```} else if (sVar == "buffersize" || sVar == "buffer") {```
to keep backwards compatibility

**Comment by jpnurmi** [comment]
:+1: 'buffer count' was very confusing, considering that there's another setting called MaxQueryBuffers that limits the count of query buffers. Anyway, the combined diff looks good. The commits could be squashed to one. Would you like to implement the same for webadmin?

**Comment by ChasedSpade** [comment]
Sure. Would you like me to submit a new pull request or just keep using this one?

**Comment by jpnurmi** [comment]
Either is fine. Whichever you prefer :)

**Comment by jpnurmi** [comment]
Thanks for the contribution! :)

**Pull request closed without merging**